devtools/create_project/msvc.cpp

```diff
@@ -59,7 +59,7 @@ std::string MSVCProvider::getLibraryFromFeature(const char *feature, const Build
 		{      "sdl2", "SDL2.lib",                  "SDL2d.lib",     "winmm.lib imm32.lib version.lib setupapi.lib",    0 },
 		{      "libz", "zlib.lib",                  "zlibd.lib",     0,                                                 0 },
 		{       "mad", "mad.lib",                   0,               0,                                                 "libmad.lib" },
-		{   "fribidi", "libfribidi.lib",            0,               0,                                                 0 },
+		{   "fribidi", "fribidi.lib",               0,               0,                                                 "libfribidi.lib" },
```

Comment:
I'm not sure if `libfribidi.lib` is needed. The `vcpkg` library is `fribidi.lib`. Opinions?

Reply:
@lephilousophe How does it work on buildbot?

Reply:
It works without create_project because we use mingw and configure script so it's not really relevant. We use `-lfribidi` and linker must find libfribidi.a.

Reply:
Ah, I see. Thanks.

So can I remove the libfribidi.lib?

Reply:
I assume vcpkg changed something that renamed libfribidi to fribidi? I'll have to update the prebuilt libraries.

Reply:
Has it ever worked?